lib/core/updateSimplifiedMasternodeListFactory.js

```
@@ -54,6 +54,9 @@ function updateSimplifiedMasternodeListFactory(
       // to be able to verify signatures
 
       const startHeight = coreHeight - smlMaxListsLimit;
+      if (startHeight < 1) {
+        return;
```

I would throw an error here otherwise further logic which relay on SML won't work.